website/documentation/050-tutorials/content/howto/maintain-shoot/_index.md

```diff
@@ -37,17 +54,45 @@ spec:
       kubernetesVersion: true
 ```
 
-If you update the CloudProfile used in the Shoot and add `1.10.5` and `1.11.0` to the `.spec.kubernetes.versions` list, the Shoot will be updated to `1.10.5` between 22:00-23:00 UTC. Your Shoot won't be updated to `1.11.0` even though its the highest Kubernetes version in the CloudProfile. This is because that wouldn't be a patch release update but a minor release update, and potentially have breaking changes that could impact your deployed resources.
+Updating the CloudProfile used in the Shoot by adding `1.10.5` and `1.11.0` to the `.spec.kubernetes.versions` list, automatically updates the Shoot to `1.10.5` between 22:00-23:00 UTC. The Shoot won't be automatically updated to `1.11.0` even though its the highest Kubernetes version in the `CloudProfile`. This is because that wouldn't be a patch release update but a minor release update, and as such potentially have breaking changes that could impact the deployed workload/applications.
 
 In this example if the operator wants to update the Kubernetes version to `1.11.0`, he/she must update the Shoot's `.spec.kubernetes.version` to `1.11.0` manually.
 
-### Kubernetes Version Expiration Date
+Besides automatic versioning during the maintenance time, version updates can also be handled by updating the `.spec.kubernetes.version` field manually. In the above example, if the operator wants to update the Kubernetes version to `1.11.0`, he/she can update the Shoot's `.spec.kubernetes.version` to `1.11.0` manually. 
+
+These updates will either be executed immediately (default) or can be [confined to the maintenance time window](https://github.com/gardener/gardener/blob/master/docs/usage/shoot_maintenance.md#confine-specification-changesupdates-roll-out).
+Choosing the latter option, causes changes to the cluster (e.g node pool rolling-updates) and the subsequent reconciliation, to only predictably happen during a defined time window. This is available since [Gardener version 1.4](https://github.com/gardener/gardener/releases/tag/v1.4.0).
```

Comment:
```suggestion
Choosing the latter option, causes changes to the cluster (e.g. node pool rolling-updates) and the subsequent reconciliation, to only predictably happen during a defined time window. This is available since [Gardener version 1.4](https://github.com/gardener/gardener/releases/tag/v1.4.0).
```